code/src/cljs/sixsq/nuvla/ui/edge_detail/views.cljs

```diff
@@ -198,6 +203,10 @@
     (fn [{:keys [id] :as resource} operation show? title icon button-text]
       (when (not= (:parent @status))
         (dispatch [::events/get-nuvlabox id]))
+      (swap! form-data assoc :project-name project)
```

Comment:
This will make project-name, working-dir, etc. not editable. 
Do you want that behavior. If yes, it's better to make input read-only and pre-filled

Reply:
Nope that's certainly not my intent. Thanks I'll fix that